#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-manpage.dpatch by Justin Pryzby <jpryzby@quoininc.com>
##
## DP: manpage typos (Closes: #461934).

@DPATCH@

diff -Naurp rdiff-backup-1.1.15.orig/rdiff-backup.1 rdiff-backup-1.1.15/rdiff-backup.1
--- rdiff-backup-1.1.15.orig/rdiff-backup.1	2008-01-03 15:36:49.000000000 +0000
+++ rdiff-backup-1.1.15/rdiff-backup.1	2008-01-21 20:37:14.000000000 +0000
@@ -366,11 +366,11 @@ If the filesystem to which we are backin
 .B \-\-preserve-numerical-ids
 If set, rdiff-backup will preserve uids/gids instead of trying to
 preserve unames and gnames.  See the
-.B USERS and GROUPS
+.B USERS AND GROUPS
 section for more information.
 .TP
 .B \-\-print-statistics
-If set, summary statistics will be printed after a successful backup
+If set, summary statistics will be printed after a successful backup.
 If not set, this information will still be available from the
 session statistics file.  See the
 .B STATISTICS
@@ -483,7 +483,7 @@ will be ignored.
 Map user names and ids according to the user mapping file
 .IR filename .
 See the
-.B USERS and GROUPS
+.B USERS AND GROUPS
 section for more information.
 .TP
 .BI \-v [0-9] ", \-\-verbosity " [0-9]
@@ -644,7 +644,7 @@ string specified by \-\-remote-schema, q
 Although ssh itself may be secure, using rdiff-backup in the default
 way presents some security risks.  For instance if the server is run
 as root, then an attacker who compromised the client could then use
-rdiff-backup to overwrite arbitary server files by "backing up" over
+rdiff-backup to overwrite arbitrary server files by "backing up" over
 them.  Such a setup can be made more secure by using the sshd
 configuration option
 .B command="rdiff-backup \-\-server"
@@ -965,7 +965,7 @@ The
 and
 .B \-\-group-mapping-file
 options override this behavior.  If either of these options is given,
-the policy descriped in 2 and 3 above will be followed, but with the
+the policy described in 2 and 3 above will be followed, but with the
 mapped user and group instead of the original.  If you specify both
 .B \-\-preserve-numerical-ids
 and one of the mapping options, the behavior is undefined.
@@ -1022,7 +1022,7 @@ directory statistics file is more compac
 but describes every directory backed up.  It also may be compressed to
 save space.
 
-Statistics related options include
+Statistics\-related options include
 .B \-\-print-statistics
 and
 .BR \-\-null-separator .
@@ -1030,7 +1030,7 @@ and
 Also, rdiff-backup will save various messages to the log file, which
 is rdiff-backup-data/backup.log for backup sessions and
 rdiff-backup-data/restore.log for restore sessions.  Generally what is
-written to this file will coincide with the messages diplayed to
+written to this file will coincide with the messages displayed to
 stdout or stderr, although this can be changed with the
 .B \-\-terminal-verbosity
 option.
